setup.py

```diff
@@ -97,7 +96,13 @@ def read(path, encoding="utf-8"):
 ### NEEDED FOR A BATTERIES-LOADED EXPERIENCE
 udf_libs = [
     "facenet-pytorch>=2.5.2",
-    "easyocr>=1.5.0"
+    "easyocr>=1.5.0",
+    "ipython",
+    # "psutil",
```

Comment:
I noticed these Tensorflow dependencies. Are you able to add a Tensorflow-based UDF?

Reply:
No Sir, `tensorboard` is a utility to monitor model performance on training and testing.
It is build to work without `tensorflow` library.